app/api/dao/user_extension.py

```diff
@@ -0,0 +1,32 @@
+from http import HTTPStatus
+from typing import Dict
+from sqlalchemy import func
+from app.database.models.bit_schema.user_extension import UserExtensionModel
+from app import messages
+
+
+class UserExtensionDAO:
+    
+    """Data Access Object for Users_Extension functionalities"""
+
+    @staticmethod
+    def create_user_extension(data):
+        """Creates a user_extension instance for a new registered user.
+
+        Arguments:
+            data: A list containing user's id, boolean value of whether or not
+            the user is representing an organization, as well as their timezone
+    
+        Returns:
+                A tuple with two elements. The first element is a dictionary containing a key 'message' containing a string which indicates whether or not the user_exension was created successfully. The second is the HTTP response code.
+        """
+
+        user_id = data["user_id"]
+        is_organization_rep = data["is_organization_rep"]
+        timezone = data["timezone"]
+
+        user_extension = UserExtensionModel(user_id, is_organization_rep, timezone)
+        
+        user_extension.sava_to_db()
```

Comment:
"save_to_db" probably? Not "sava"

Reply:
😂 😂 😂  Thanks for pointing this out, @ramitsawhney27 . will fix that